WordPress/src/main/java/org/wordpress/android/models/Person.java

```diff
@@ -50,9 +53,9 @@ public static Person fromJSON(JSONObject json, int localTableBlogId) {
             String displayName = json.optString("nice_name");
             String avatarUrl = json.optString("avatar_URL");
             // We don't support multiple roles, so the first role is picked just as it's in Calypso
-            Role role = Role.fromKey(json.optJSONArray("roles").optString(0));
+            String role = json.getJSONArray("roles").optString(0);
 
-            return new Person(personID, localTableBlogId, username, firstName, lastName, displayName, avatarUrl, role);
+            return new Person(personID, siteID, localTableBlogId, username, firstName, lastName, displayName, avatarUrl, role);
```

Comment:
Let's wrap this line to 180chars. Cheers!
While at it, there's a comment line a few lines above that needs wrapping as well.